rail_scripts/rail-preprocess-parquet

```diff
@@ -181,14 +187,15 @@ def preprocess(cfg):
                 print(input_)
                 return
 
-            has_magnitude, has_flux, has_ra, has_dec = parquet_check_column_presence(
+            has_magnitude, has_flux, has_ra, \
+            has_dec, has_objectId = parquet_check_column_presence(
                     f, flux_columns + flux_error_columns,
                     mag_columns + mag_error_columns, 
-                    ra_column, dec_column)
+                    ra_column, dec_column, objectId_column)
             
             use_flux, coord_columns, value_columns, error_columns = choose_columns(
-                    has_ra, has_dec, has_magnitude, has_flux, 
-                    cfg.ra, cfg.dec, cfg.flux, flux_columns, 
+                    has_ra, has_dec, has_objectId, has_magnitude, has_flux, 
+                    cfg.ra, cfg.dec, cfg.objectId, cfg.flux, flux_columns, 
```

Comment:
Inconsistent coding style: please use underscore notation for variable names: `cfg.object_id`.